tests/tests_integration/test_api/test_geospatial.py

```diff
@@ -706,8 +707,8 @@ def test_put_raster_custom_crs(self, cognite_client, test_feature_type, test_fea
         assert res.skew_x == 0.0
         assert res.skew_y == 0.0
         assert res.srid == 3857
-        assert res.upper_left_x == -0.5891363261459447
-        assert res.upper_left_y == -0.31623471547260973
+        assert math.isclose(res.upper_left_x, -0.5891363261459447)
+        assert math.isclose(res.upper_left_y, -0.31623471547260973, rel_tol=5e-7)
```

Comment:
we could change the expected value to the new one: -0.316234582133065 and use the default tolerance.

Reply:
the value is gotten from running test upon master branch

Reply:
Why has the API response changed? If this is because the underlying test data has changed, I’ll happily update to a new value. But, if this is a calculated value that just suddenly changed, I’m quite worried tbh 😅